tools/site_and_regional/subset_data

```diff
@@ -7,16 +7,30 @@ python/ctsm folder.
 
 For full instructions on how to run the code and different options,
 please check python/ctsm/subset_data.py file.
-
 This script extracts domain files, surface dataset, and DATM files
 at either a single point or a region using the global dataset.
+<<<<<<< HEAD
```

Comment:
Hi Adrianna, 
I think this was a merge conflict here but I can resolve it on the branch. (just FYI)
Thanks!
Negin